src/bin/commands/new-project/utils/utils.ts

```diff
@@ -28,7 +28,7 @@ export function getCommands(
   return {
     installDependencies: () => runTask(packageManager, cwd, "install"),
     lint: () => runTask(packageManager, cwd, "run lint"),
-    test: () => runTask(packageManager, cwd, "test"),
+    test: () => runTask(packageManager, cwd, "test -- -u"),
```

Comment:
See https://github.com/guardian/cdk/pull/1392/files#r925215363 and https://github.com/facebook/jest/pull/12378.